akka-remote/src/main/protobuf/WireFormats.proto

```diff
@@ -101,6 +101,32 @@ message DeployData {
  * Akka Protocol message formats
  ******************************************/
 
+/**
+ * Defines a remote address, for classic remoting.
+ *
+ * Note that while the same message definition exists in cluster, as well as an updated one exists for Artery.
+ * This encoding will ONLY be used when an `akka.actor.Address` is attempted to be serialized *directly*.
+ */
+message ClassicAddress {
+  // ... since `Address` is already taken in this package by Artery's Address (which changes order slightly)
+  required string system = 1;
+  required string hostname = 2;
+  required uint32 port = 3;
+  optional string protocol = 4;
+}
+
+/**
+ * Defines a 'raw' remote address with uid.
+ * This encoding will ONLY be used when an `akka.remote.UniqueAddress` is attempted to be serialized *directly*.
+ */
+message ClassicUniqueAddress {
+  // ... since `UniqueAddressAddress` is already taken in this package by Artery's Address (which compacts uid into one field order slightly)
+  required ClassicAddress address = 1;
+  required uint32 uid = 2;
+  // 64 bit uids but with backward wire compatibility
+  optional uint32 uid2 = 3;
+}
+
```

Comment:
to explain the comments see: https://github.com/akka/akka/blob/master/akka-remote/src/main/protobuf/ArteryControlFormats.proto#L64-L80

So the protobuf package `akka.remote` already contains Address and UniqueAddress, however they have a different format. Note also that this proto file even contains AddressData, which could be used for the ClassicAddress I guess. I did try if we could rename the ClassicAddress but seems that's not legal, renaming field names is tho, seems message names not as much?

Reply:
Removed the duplication of ClassicAddress vs AddressData, I guess that's preferred

Reply:
ClusterMessages.proto:
```
message Address {
  required string system = 1;
  required string hostname = 2;
  required uint32 port = 3;
  optional string protocol = 4;
}

message UniqueAddress {
  required Address address = 1;
  required uint32 uid = 2;
  // 64 bit uids but with backward wire compatibility
  optional uint32 uid2 = 3;
}
```

ArteryControlFormats.proto:
```
message Address {
  required string protocol = 1;
  required string system = 2;
  required string hostname = 3;
  required uint32 port = 4;
}

message UniqueAddress {
  required Address address = 1;
  required uint64 uid = 2;
}
```

Since this is a new format it doesn't have to be the same as the ClusterMessages.
Why can't we use the ArteryControlFormats instead, and use those classes in the MiscSerializer?

Reply:
Was perhaps over thinking the compatibility there, will use the existing protos